content/docs/dataform-web/guides/environments.md

```diff
@@ -145,3 +145,40 @@ An alternative approach to separating production and staging data is to append a
   ]
 }
 ```
+
+## Example: separating development and production data into two databases
+
+Some teams may not be at the stage where they require a `staging` environment, but still would like to keep development and production data separated. This can be done using a `configOverride` in the `production` environment.
+
+In the example below:
+
+- any code deployed whilst developing will use the `defaultDatabase` from the `dataform.json`
+- any code deployed by schedules will be in the `production` environment and so use the `defaultDatabase` from the `configOverride` in `environmemts.json`
```

Comment:
suggest rewording these last two bullets. they make it sounds like a schedule <-> environment linkage is sort of dynamic, or that you might be able to configure it, but that's not the case. i suggest something like:

```suggestion
- schedules only run in the single `production` environment, and so use the `defaultDatabase` from that environment's `configOverride`
- the `production` environment specifies the `master` Git branch, so all of its schedules will run using that version of the code
```